### DISCUSSION TITLE
How can I tell if dosemu is using SDL?

### DISCUSSION THREAD

**Discussion by decuser** [dosemu2/dosemu2#2119] [General]
Really, I have several related questions along these lines.

I'm using dosemu to run wordstar and I'd like to optimize the experience visually. I have tried messing with various settings to see if I can improve the display and some changes seem to work, others appear to have no effect.

I tried `dosemu -S` and could not see any difference. I expected to see some controls on the basis that the configuration settings in the conf file show:

> # Show window controls.
> # It is not a common need to disable them, and can be done at run-time
> # by pressing Ctrl-Alt-b.
> # Default: on

I tried changing the columns and rows in the text console `$_fixed_term_size = "132x50"` and other values to no avail.

Any tips appreciated.

**Comment by decuser** [comment]
I'm running it on Linux Mint 21.2 Victoria in Cinnamon, btw.

**Comment by stsp** [comment]
What are you trying to get?
sdl is long ago a default, and term
size applies to terminal mode.
So no wonder all you do, makes no
change.
The only thing you may change, is to
enable ttf fonts and use scaling filters,
but I think they are on by default anyway.

**Comment by decuser** [comment]
hi Stas. I'd like the terminal to be 100x40 or 132x50, or pretty much anything greater than the default 80x25.

**Reply by stsp** [reply]
Then use `dosemu -t`.

**Reply by ecm-pushbx** [reply]
Have you tried `xmode` command in the dosemu2 window? I think that can change the displayed size, not sure.
